### ISSUE TITLE
Full text reindexing doesn't work

### ISSUE THREAD

**Issue by idlefella** [blazegraph/database#7]
Hi everybody
I was experimenting with the full text index of blazegraph and I've determined that the full text index doesn't work anymore after a reindex.
Steps to reproduce:

- Start the blazegrapgh v2.1 server by `java -server -Xmx4g -jar blazegraph.jar`
- Create a new namespace **test** and activate full text search. My config looks like this: 
```com.bigdata.rdf.store.AbstractTripleStore.textIndex=true
com.bigdata.rdf.store.AbstractTripleStore.axiomsClass=com.bigdata.rdf.axioms.NoAxioms
com.bigdata.rdf.sail.isolatableIndices=false
com.bigdata.rdf.store.AbstractTripleStore.justify=false
com.bigdata.rdf.sail.truthMaintenance=false
com.bigdata.rdf.sail.namespace=test
com.bigdata.rdf.store.AbstractTripleStore.quads=false
com.bigdata.namespace.test.spo.com.bigdata.btree.BTree.branchingFactor=1024
com.bigdata.namespace.test.lex.com.bigdata.btree.BTree.branchingFactor=400
com.bigdata.journal.Journal.groupCommit=false
com.bigdata.rdf.store.AbstractTripleStore.geoSpatial=false
com.bigdata.rdf.store.AbstractTripleStore.statementIdentifiers=false
```
- Go to the update tab and insert some dummy data, e.g.:
```
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
INSERT DATA {
	<http://example.org/test> rdfs:label "Test property" .
}
```
- When you query now the data with full text search, you will find a result:
```
PREFIX bds: <http://www.bigdata.com/rdf/search#>

SELECT * {
  ?o bds:search "test*" .
}
```
- If you rebuild the full text index with the button on the namespaces tab (or recreate it in the console), the full text search doesn't work anymore, e.g. the same query as in the point above doesn't return any values.

Best idlefella

**Comment by beebs-systap** [comment]
It looks like you may have hit https://jira.blazegraph.com/browse/BLZG-1893.
  There's a fix in 2.1.1, which will be out very shortly.

On Mon, May 23, 2016 at 11:30 AM, idlefella <notifications@github.com>
wrote:

> Hi everybody
> I was experimenting with the full text index of blazegraph and I've
> determined that the full text index doesn't work anymore after a reindex.
> Steps to reproduce:
>
>    - Start the blazegrapgh v2.1 server by java -server -Xmx4g -jar
>    blazegraph.jar
>    - Create a new namespace *test* and activate full text search. My
>    config looks like this:
>
> com.bigdata.rdf.store.AbstractTripleStore.axiomsClass=com.bigdata.rdf.axioms.NoAxioms
> com.bigdata.rdf.sail.isolatableIndices=false
> com.bigdata.rdf.store.AbstractTripleStore.justify=false
> com.bigdata.rdf.sail.truthMaintenance=false
> com.bigdata.rdf.sail.namespace=test
> com.bigdata.rdf.store.AbstractTripleStore.quads=false
> com.bigdata.namespace.test.spo.com.bigdata.btree.BTree.branchingFactor=1024
> com.bigdata.namespace.test.lex.com.bigdata.btree.BTree.branchingFactor=400
> com.bigdata.journal.Journal.groupCommit=false
> com.bigdata.rdf.store.AbstractTripleStore.geoSpatial=false
> com.bigdata.rdf.store.AbstractTripleStore.statementIdentifiers=false
>
>
>    - Go to the update tab and insert some dummy data, e.g.:
>
> PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
> INSERT DATA {
>     <http://example.org/test> rdfs:label "Test property" .
> }
>
>
>    - When you query now the data with full text search, you will find a
>    result:
>
> PREFIX bds: <http://www.bigdata.com/rdf/search#>
>
> SELECT * {
>   ?o bds:search "test*" .
> }
>
>
>    - If you rebuild the full text index with the button on the namespaces
>    tab (or recreate it in the console), the full text search doesn't work
>    anymore, e.g. the same query as in the point above doesn't return any
>    values.
>
> Best idlefella
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/blazegraph/database/issues/7>
>



-- 
_______________
Brad Bebee
CEO
Blazegraph
e:  beebs@blazegraph.com
m: 202.642.7961
w:  www.blazegraph.com

Blazegraph products help to solve the Graph Cache Thrash to achieve large
scale processing for graph and predictive analytics.  Blazegraph is the
creator of the industry’s first GPU-accelerated high-performance database
for large graphs, has been named as one of the “10 Companies and
Technologies to Watch in 2016” <http://insideanalysis.com/2016/01/20535/>.

Blazegraph Database <https://www.blazegraph.com/> is our ultra-high
performance graph database that supports both RDF/SPARQL and Apache
TinkerPop™ APIs.   Blazegraph GPU
<https://www.blazegraph.com/product/gpu-accelerated/> andBlazegraph DAS
<https://www.blazegraph.com/product/gpu-accelerated/>L are disruptive new
technologies that use GPUs to enable extreme scaling that is thousands of
times faster and 40 times more affordable than CPU-based solutions.

CONFIDENTIALITY NOTICE:  This email and its contents and attachments are
for the sole use of the intended recipient(s) and are confidential or
proprietary to SYSTAP, LLC DBA Blazegraph.  Any unauthorized review, use,
disclosure, dissemination or copying of this email or its contents or
attachments is prohibited. If you have received this communication in
error, please notify the sender by reply email and permanently delete all
copies of the email and its contents and attachments.

**Comment by idlefella** [comment]
Yes I think it's exactly the same.
